addons/mail/static/src/components/messaging_menu/messaging_menu.scss

```diff
@@ -37,7 +37,7 @@
         * As a result, Min() is ignored by SCSS while CSS interprets as its min() function.
         */
         max-height: Min(calc(100vh - 140px), 630px);
-        z-index: 1100; // on top of chat windows
+        z-index: 1050; // on top of chat windows
```

Comment:
Has this been validated by a designer ?

Reply:
z-index principle is such a mess that it's hard to know for sure, but comment says "on top of chat windows", and chat window have z-index 1000, so this is ok to me.
If it was supposed to be above something else too, it should be been commented as well.